vaadin-cdi/src/test/java/com/vaadin/cdi/internal/MyBean.java

```diff
@@ -10,7 +9,6 @@
     private final int id = counter++;
 
     public MyBean() {
-        System.out.println(UI.getCurrent());
```

Comment:
It would be good to understand why this causes an NPE, but that doesn't block merging these changes.